src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -278,10 +385,10 @@ class GroupReadsByUmi
     case "identity"  => new IdentityUmiAssigner
     case "edit"      => new SimpleErrorUmiAssigner(edits)
     case "adjacency" => new AdjacencyUmiAssigner(edits)
+    case "paired"    => new PairedUmiAssigner(edits)
```

Comment:
I like having it all in one file and making a Java enum forces me to break it apart.  I'm going to leave it for now unless you feel super strongly about this.